DuggaSys/sectioned.js

```diff
@@ -3736,12 +3736,14 @@ function quickValidateForm(formid, submitButton) {
   }
 
   if (formid === 'editSection') {
-    var sName = document.getElementById("sectionname").value;
-    var item = document.getElementById("editSectionDialogTitle").innerHTML;
-    var deadlinepart = document.getElementById('inputwrapper-deadline');
-    var deadlinedisplayattribute = deadlinepart.style.display;
+    let sName = document.getElementById("sectionname").value;
+    let item = document.getElementById("editSectionDialogTitle").innerHTML;
+    let feedback = document.getElementById("courseFeedback").value;
+    let deadlinepart = document.getElementById('inputwrapper-deadline');
+    let deadlinedisplayattribute = deadlinepart.style.display;
     valid = true;
     valid &= validateSectName('sectionname');
+    valid &= validateSectName('courseFeedback');
```

Comment:
The 'validate section name' (`validateSectName`) function cannot be reused as it is connected to a specific error dialog. You have to create-new/copy the function and a create a new dialog element to reference